test/bats/test.bats

```diff
@@ -278,10 +278,26 @@ __namespace_exclusion_test() {
     skip "skipping external data tests"
   fi
 
-  # deployment, service and provider for dummy-provider
-  run kubectl apply -f test/externaldata/dummy-provider/manifest
+  if [ ! -f test/externaldata/dummy-provider/certs/ca.crt ]; then
+    echo "Missing dummy-provider's CA cert. Please run test/externaldata/dummy-provider/scripts/generate-tls-certificate.sh to generate it."
```

Comment:
If we have a tmp folder, can we not just auto-generate the cert there and avoid requiring users to prep before testing?

Reply:
We need the cert as part of the image-building step of the dummy provider so we can load it into the image (see `make e2e-build-load-externaldata-image`. It'd be too late if we were to generate the cert as part of the test step.

Reply:
Makes sense